### DISCUSSION TITLE
workflow logs

### DISCUSSION THREAD

**Discussion by pocelka** [github/vscode-github-actions#296] [Q&A] [answered]
Hi all,

Could anyone help me on how can I see logs from github action execution? I see this:
![image](https://github.com/github/vscode-github-actions/assets/20441822/fe3bb1d6-5fc7-45ef-b768-759fd5585d16)

However as per extension screenshots I should be able to see logs. Right now I can see only icon to go to directly to github actions; i.e. hamburger menu is missing.

Do I need to set some configuration to see them?

**Comment by muzimuzhi** [comment, marked as answer]
Hover on the job name and click the [list-unordered icon](https://code.visualstudio.com/api/references/icons-in-labels#icon-listing), you will have the full job log opened in a new tab.
![image](https://github.com/github/vscode-github-actions/assets/6376638/42333db1-4d45-42b1-85fe-cd1c0e271eb8)

<img width="952" alt="image" src="https://github.com/github/vscode-github-actions/assets/6376638/28e2fa89-f62a-4560-818e-23511ebec4f2">
<img width="925" alt="image" src="https://github.com/github/vscode-github-actions/assets/6376638/6f89817e-42c4-4573-8ad6-19fa8a4b608a">
The log is light highlighted and the only interaction is that every group is collapsable.

It seems with the current [v0.26.2](https://github.com/github/vscode-github-actions/releases/tag/release-v0.26.2), there's no support for jumping to start of a specific step in a job log.

**Comment by pocelka** [comment]
I feel really stupid since I was expecting to see the individual logs for each step instead for the whole run. 

Thank you for pointing out that I was looking on it wrongly.
